Power BI Connector/OSDUWellsConnector/OSDUWellsConnector/OSDUWellsConnector.pq

```diff
@@ -32,7 +32,7 @@ shared OSDUWellsConnector.Contents = (kind as text, query as text, optional limi
              ]  
            ))
     in
-        Source[results];
```

Comment:
Is this change required? It will break existing reports.

Reply:
Yes, this change is quite useful to retrieve counts for volume dashboards. Also I am not aware of any present customer usage and we can publish future templates based on the new version.